src/views/forms/TransferForm.tsx

```diff
@@ -61,58 +61,67 @@ export const TransferForm = ({
   const stringGetter = useStringGetter();
   const { freeCollateral } = useSelector(getSubaccount, shallowEqual) || {};
   const { dydxAddress } = useAccounts();
-  const {
-    address: recipientAddress,
-    size,
-    fee,
-  } = useSelector(getTransferInputs, shallowEqual) || {};
   const { transfer } = useSubaccount();
   const { nativeTokenBalance, usdcBalance } = useAccountBalance();
   const { selectedNetwork } = useSelectedNetwork();
   const { tokensConfigs, usdcLabel, chainTokenLabel } = useTokenConfigs();
 
-  // User Input
-  const [asset, setAsset] = useState<DydxChainAsset>(selectedAsset);
+  const {
+    address: recipientAddress,
+    size,
+    fee,
+    token,
+  } = useSelector(getTransferInputs, shallowEqual) || {};
 
   // Form states
   const [error, setError] = useState<string>();
   const [isLoading, setIsLoading] = useState(false);
 
-  const balance = asset === DydxChainAsset.USDC ? freeCollateral?.current : nativeTokenBalance;
-  const newBalance =
-    asset === DydxChainAsset.USDC
-      ? freeCollateral?.postOrder
-      : MustBigNumber(nativeTokenBalance)
-          .minus(size?.size ?? 0)
-          .toNumber();
-  const amount = asset === DydxChainAsset.USDC ? size?.usdcSize : size?.size;
+  // temp fix: TODO: reset fees when changing token in Abacus
```

Comment:
is this `TODO` already handled in `onChangeAsset`?

Reply:
ah as in I added the fees reset in `onChangeAsset`, but the actual fix should be in Abacus